lib/logstash/outputs/datadog_logs.rb

```diff
@@ -215,7 +216,11 @@ def initialize(logger, use_ssl, no_ssl_validation, host, port, use_compression,
       logger.info("Starting HTTP connection to #{protocol}://#{host}:#{port.to_s} with compression " + (use_compression ? "enabled" : "disabled") + (force_v1_routes ? " using v1 routes" : " using v2 routes"))
 
       config = {}
-      config[:ssl][:verify] = :disable if no_ssl_validation
+      config[:ssl] = {}
+      config[:verify] = :disable if no_ssl_validation
```

Comment:
This change of `[:verify]` level is expected?

I.e. if I'm right we're not setting `config.ssl.verify` anymore but `config.verify`?

Wasn't previous L218 crashing because `config[:ssl]` undefined?!